src/Tmds.LinuxAsync/IOUringAsyncEngine.IOUringThread.cs

```diff
@@ -148,14 +148,15 @@ public unsafe void Post(Action<object?> action, object? state)
 
                 if (blockingState == StateBlocked)
                 {
-                    WriteToPipe();
+                    WriteToEventFd();
                 }
             }
 
-            private unsafe void WriteToPipe()
+            private unsafe void WriteToEventFd()
             {
                 Span<byte> buffer = stackalloc byte[8];
-                int rv = IoPal.Write(_pipeWriteEnd!, buffer);
+                Unsafe.WriteUnaligned(ref MemoryMarshal.GetReference(buffer), 1UL);
```

Comment:
`buffer[7]=1;`

Reply:
Wouldn't that be incorrect on big endian machines?

Reply:
It doesn't matter what the value is. It needs to make the eventfd readable, which happens when the counter is greater than zero.

Reply:
Pragmatic :+1: